front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -94,10 +94,15 @@ export const stdcmConfSlice = createSlice({
     },
     updateStdcmPathStep(
       state: Draft<OsrdStdcmConfState>,
-      action: PayloadAction<ArrayElement<OsrdStdcmConfState['stdcmPathSteps']>>
+      action: PayloadAction<{
+        id: string;
+        updates: Partial<ArrayElement<OsrdStdcmConfState['stdcmPathSteps']>>;
```

Comment:
If we wanted to, we could have a flatter data structure with something like:

```ts
action: PayloadAction<{ id: string } & Partial<…>>
```

But the status quo seems perfectly fine to me as well (feel free to ignore).

Reply:
Good idea, but I prefer to keep the id and the rest separated :)